monitoring/api/AlertTestFixture/AlertTestFixture.cs

```diff
@@ -0,0 +1,114 @@
+            UserLabels =
+                {
+                    { "role", "operations" },
+                    { "level", "5" },
+                    { "office", "california_westcoast_usa" },
+                    { "division", "fulfillment"}
+                }
+        };
+        return NotificationChannelClient.CreateNotificationChannel(
+            new ProjectName(ProjectId), channel);
+    }
+
+    public AlertTestFixture()
+    {
+        Channel = CreateChannel();
+
+        Alert = AlertPolicyClient.CreateAlertPolicy(
+            new ProjectName(ProjectId), new AlertPolicy()
+            {
+                DisplayName = "AlertTest.cs",
+                Enabled = false,
+                Combiner = ConditionCombinerType.Or,
+                Conditions =
+            {
+                    new AlertPolicy.Types.Condition()
+                    {
+                        ConditionThreshold = new MetricThreshold()
+                        {
+                            Filter = "metric.label.state=\"blocked\" AND metric.type=\"agent.googleapis.com/processes/count_by_state\"  AND resource.type=\"gce_instance\"",
+                            Aggregations = {
+                                new Aggregation() {
+                                    AlignmentPeriod = Duration.FromTimeSpan(
+                                        TimeSpan.FromSeconds(60)),
+                                    PerSeriesAligner = Aligner.AlignMean,
+                                    CrossSeriesReducer = Reducer.ReduceMean,
+                                    GroupByFields = {
+                                        "project",
+                                        "resource.label.instance_id",
+                                        "resource.label.zone"
+                                    }
+                                }
+                            },
+                            DenominatorFilter = "",
+                            DenominatorAggregations = {},
+                            Comparison = ComparisonType.ComparisonGt,
+                            ThresholdValue = 100.0,
+                            Duration = Duration.FromTimeSpan(
+                                TimeSpan.FromSeconds(900)),
+                            Trigger = new Trigger() {
+                                Count = 1,
+                                Percent = 0.0,
+                            }
+                        },
+                        DisplayName = "AlertTest.cs",
+                    }
+            },
+            });
+    }
+
+    public NotificationChannel Channel { get; private set; }
+    public AlertPolicy Alert { get; private set; }
+
+    public string ProjectId { get; private set; } =
+        Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+
+    public NotificationChannelServiceClient NotificationChannelClient
+    { get; private set; } = NotificationChannelServiceClient.Create();

```

Comment:
nit: { get; private set; } part should be on the same line as the variable

Reply:
I'd like to put them on the same line, but it doesn't fit in less than 80 characters.